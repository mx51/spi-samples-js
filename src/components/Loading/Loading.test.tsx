import React from 'react';
import { cleanup, render } from '@testing-library/react';
import Loading from '.';

describe('Test <Loading />', () => {
  afterEach(cleanup);

  test('should match Loading component snapshot test', () => {
    // Arrange
    const { container } = render(<Loading />);

    // Assert
    expect(container).toMatchSnapshot();
  });
});
