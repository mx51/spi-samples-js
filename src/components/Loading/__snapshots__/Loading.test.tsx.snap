// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Loading /> should match Loading component snapshot test 1`] = `
<div>
  <div>
    Loading ..
  </div>
</div>
`;
